packages/cli/src/commands/open.ts

```diff
@@ -1,20 +1,30 @@
+import type { FullConfig } from "@allurereport/core";
 import { readConfig } from "@allurereport/core";
 import { serve } from "@allurereport/static-server";
 import { Command, Option } from "clipanion";
+import { glob } from "glob";
+import { randomUUID } from "node:crypto";
+import { mkdir } from "node:fs/promises";
+import { join } from "node:path";
+import { cwd as processCwd } from "node:process";
+import { generate } from "./commons/generate.js";
 
 export class OpenCommand extends Command {
-  static paths = [["open"]];
+  static paths = [["open"], ["serve"]];
 
   static usage = Command.Usage({
     description: "Serves specified directory",
-    details: "This command serves the specified report directory and opens it in the default browser.",
+    details: "This command generates report with the given test results and opens it in the default browser.",
     examples: [
-      ["open", "Serve the default report directory"],
-      ["open custom-report --port 8080 --live", "Serve the custom-report directory on port 8080 with live reload"],
+      ["open ./allure-results", "Generate and serve the report based on given test results directory"],
+      ["open --port 8080 ./allure-report", "Serve the report on port 8080"],
     ],
   });
 
-  reportDir = Option.String({ required: false, name: "The directory with Allure results" });
+  resultsDir = Option.String({
+    required: true,
```

Comment:
`allure open` with no arguments should serve `allure-report` by default (just like it currently does)